raiden-ts/src/transfers/epics/locked.ts

```diff
@@ -131,20 +131,36 @@ function makeAndSignTransfer$(
     'expiration too soon',
   );
 
-  // fee is added to the lock amount, check for overflow
-  const amountWithFee = decode(
-    UInt(32),
-    action.payload.value.add(fee),
-    'overflow of locked amount with fee',
-  );
+  // fee is added to the lock amount; overflow is checked on locksSum below
+  const amountWithFee = action.payload.value.add(fee) as UInt<32>;
```

Comment:
total nit, but I'd group the expiration checks together and move the amount down.